gradio/interface.py

```diff
@@ -571,23 +572,18 @@ def clean_html(raw_html):
                     status_tracker=status_tracker,
                 )
             clear_btn.click(
-                (
-                    lambda: [
-                        component.default_value
-                        if hasattr(component, "default_value")
-                        else None
-                        for component in self.input_components + self.output_components
-                    ]
-                    + [True]
+                f"""() => {json.dumps(
```

Comment:
Why did you change the logic to go from `default_value` to `cleared_value`? How will this impact components like sliders that have a default value but not a clear value?

Reply:
Actually it seems to work for slider (resets to default value) but I don't know why lol

Reply:
Because some components, like a textbox, should clear when clear is pressed, not reset to the default value. For components like Slider, the cleared_value is the default value.